### PR TITLE
Adds pdbs (and other files) to testing package

### PR DESCRIPTION
**Pull request by jkotalik**
For #894

### CODE REVIEW

**Comment by pakrym** [comment]
Do we need all the other stuff?

**Comment by jkotalik** [comment]
We don't, I just wanted the quick fix. It's only used for testing anyways.

**Comment by pakrym** [comment]
Let's put just the things we need there.

File: nuget/AspNetCoreV1.nuspec [hunk 1]
```diff
@@ -19,8 +19,8 @@
     </contentFiles>
   </metadata>
   <files>
-    <file src="src\AspNetCoreModuleV1\AspNetCore\bin\$Configuration$\Win32\aspnetcore.dll" target="contentFiles\any\any\x86\aspnetcore.dll" />
-    <file src="src\AspNetCoreModuleV1\AspNetCore\bin\$Configuration$\x64\aspnetcore.dll" target="contentFiles\any\any\x64\aspnetcore.dll" />
+    <file src="src\AspNetCoreModuleV1\AspNetCore\bin\$Configuration$\Win32\**" target="contentFiles\any\any\x86\" />
```
Comment (pakrym) [inline nuget/AspNetCoreV1.nuspec]: here too

**Comment by jkotalik** [comment]
If you are referring to this, then yes: https://github.com/aspnet/IISIntegration/blob/dev/nuget/Microsoft.AspNetCore.AspNetCoreModule.props

**Pull request merged**